src/Components/Blazor/Validation/src/ValidateComplexTypeAttribute.cs

```diff
@@ -19,11 +19,10 @@ public sealed class ValidateComplexTypeAttribute : ValidationAttribute
         /// <inheritdoc />
         protected override ValidationResult IsValid(object value, ValidationContext validationContext)
         {
-            if (!ObjectGraphDataAnnotationsValidator.TryValidateRecursive(value, validationContext))
-            {
-                throw new InvalidOperationException($"{nameof(ValidateComplexTypeAttribute)} can only used with {nameof(ObjectGraphDataAnnotationsValidator)}.");
-            }
+            ObjectGraphDataAnnotationsValidator.TryValidateRecursive(value, validationContext);
```

Comment:
So currently this returns `false`, if the passed in context is not the "right" context? What does actually "right" context mean?

Reply:
Also, this is a behavior change, is this BlazorWASM?

Reply:
This is part of the experimental package. The problem with the current code is that if you were to use a type between MVC (web API) and Blazor, the validation in MVC will throw when it encounters this attribute.